app/src/test/java/com/duckduckgo/tabs/model/TabDataRepositoryTest.kt

```diff
@@ -468,7 +485,7 @@ class TabDataRepositoryTest {
     @Test
     fun getActiveTabCountReturnsCorrectCountWhenTabsYoungerThanSpecifiedDay() = runTest {
```

Comment:
@0nko after fixing the flaky test, this one is now failing. Can you evaluate the expected result?